types/chart.js/index.d.ts

```diff
@@ -313,7 +315,7 @@ declare namespace Chart {
         callbacks?: ChartTooltipCallback;
         filter?(item: ChartTooltipItem, data: ChartData): boolean;
         itemSort?(itemA: ChartTooltipItem, itemB: ChartTooltipItem): number;
-        position?: string;
+        position?: TooltipPositionType;
```

Comment:
This doesn't seem to be true, given the [documentation](https://www.chartjs.org/docs/latest/configuration/tooltip.html#position-modes).
This change will cause errors for any custom defined modes from `positioners`.
@lekoaf, did I misunderstand something?

Reply:
@uniqueiniquity I changed the type to support custom positioners. But I am not sure the PR still relevant.

Reply:
I agree; this will just be `string` in the end. I'd suggest closing the PR.